{"name": "Verify error message for login with incorrect credentials -- @1.1 ", "status": "passed", "steps": [{"name": "Given the user is on the login page", "status": "passed", "start": 1751360145828, "stop": 1751360146572}, {"name": "When the user enter the standard_user and invalid_password", "status": "passed", "start": 1751360146573, "stop": 1751360146689}, {"name": "Then the Epic sadface: Username and password do not match any user in this service is displayed", "status": "passed", "start": 1751360146689, "stop": 1751360146701}], "parameters": [{"name": "username", "value": "standard_user"}, {"name": "password", "value": "invalid_password"}, {"name": "error message", "value": "Epic sadface: Username and password do not match any user in this service"}], "start": 1751360145692, "stop": 1751360146749, "uuid": "ba0d6817-785e-4340-a5a8-3410be89c8c7", "historyId": "28f909ecd75e3205851eff6c17944d9b", "fullName": "User login functionality: Verify error message for login with incorrect credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "invalid_user"}, {"name": "tag", "value": "login"}, {"name": "feature", "value": "User login functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}